README.md

```diff
@@ -262,6 +262,7 @@ The following is a list of other *OPTIONAL* props you may provide to the `DayPic
   renderCalendarInfo: PropTypes.func,
   onOutsideClick: PropTypes.func,
   keepOpenOnDateSelect: PropTypes.bool,
+  borderless: PropTypes.bool,
```

Comment:
I generally like to name props like this `noFoo` instead of `fooless` because I think it is easier to be more consistent that way.

Reply:
sounds reasonable to me!